test/E2ETests/E2EApps/E2EAspNetCoreApp/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="AzureFunctionsTempStaging" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsTempStaging/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
```

Comment:
do we need this nuget config? it should inherit from parent directories... (unless this is adding something new)

Reply:
The "setup-e2e-tests" script adds a local nuget repo with the latest built worker/sdk so I believe this file is needed.